bashrc.d/docker.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+# exec a command in a running container: dr-cmd abcd1234 cmd...
+dr-exec() {
+  docker exec -it $1 "${@:2}"
+}
+
+# exec a command using fzf to target a running container: dr-cmd  cmd...
+drf-exec() {
+  containerId=$(docker ps | egrep -v 'CONTAINER ID' | fzf | cut -d" " -f1)
```

Comment:
This line complained about 'egrep' and suggested 'grep -E' instead

Notice: ./bashrc.d/docker.sh:9:29: note: egrep is non-standard and deprecated. Use grep -E instead. [SC2196]